src/feature/hs/feature_hs.dox

```diff
@@ -1,4 +1,5 @@
 /**
 @dir /feature/hs
-@brief feature/hs
+@brief feature/hs: v3 (current) onion service protocol
```

Comment:
I would be super greatful if you would put here a ref to the `rend-spec-v3.txt` :D like it was done above:

[dir-spec.txt](https://gitweb.torproject.org/torspec.git/tree/dir-spec.txt),